docs/streamingapi/reference/reference.md

```diff
@@ -220,6 +220,31 @@ setTimeout(() => {
 }
 ```
 
+### Modify Request​
+This is how to modify the request during a conversation using the Streaming API. If there is a device change event during the conversation after the WebSocket connection is established. 
```

Comment:
@amritesh-singh 
Rewrite to: 

"The `modify_request` allows you to modify the request during Streaming API call, i.e., after the WebSocket connection has established, if there is a device change event.